docs/section10/10-14.md

```diff
@@ -0,0 +1,177 @@
+# Airside systems – Rule 10-14  
+**Schema Version:** 0.0.36        
+**Mandatory Rule:** True          
+**Rule ID:** 10-14  
+ 
+**Rule Description:** Baseline shall be modeled with the heating HVAC system efficiency per Tables G3.5.1-G3.5.6 (applies only to the heating efficiency of baseline furnaces and heat pumps).  Where multiple HVAC zones or residential spaces are combined into a single thermal block the heating efficiencies (for baseline HVAC System Types 3 and 4) shall be based on the  equipment capacity of the thermal block divided by the number of HVAC zones or residential spaces.   
+
+**Rule Assertion:** B-RMR = expected value                                           
+**Appendix G Section:** Section G3.1.2.1 Equipment Efficiencies  
+**Appendix G Section Reference:** None  
+**Data Lookup:** Tables G3.5.2, G3.5.4 and G3.5.5  
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem   
+
+**Applicability Checks:**  
+
+1. Applicable to baseline HVAC system types 2, 3, 3a, 4, and 9.  
+
+**Function Call:**  
+
+1. get_baseline_system_types()  
+2. get_hvac_zone_list_w_area()  
+3. baseline_system_type_compare()  
+
+## Rule Logic:   
+**Applicability Check 1:**  
+- Get baseline hvac system type dictionary: `baseline_hvac_sys_type_ids_dict_b = get_baseline_system_types(B_RMR)`
+- Create a list of the target system types: `target_system_types = [HVAC_SYS.SYS_2, HVAC_SYS.SYS_3, HVAC_SYS.SYS_3a, HVAC_SYS.SYS_4, HVAC_SYS.SYS_9]`   
+- For each hvac system in the B_RMR: `for hvac_b in B_RMR...HeatingVentilationAirConditioningSystem:`  
+    Check if the hvac system is applicable to this check based on system type  
+    - Get the hvac system type for hvac_b: `hvac_sys_type_b = baseline_hvac_sys_type_ids_dict_b.keys()[list(baseline_hvac_sys_type_ids_dict_b.values()).index(hvac_b.id)]`
```

Comment:
`hvac_sys_type_b = next((sys_type for sys_type, hvac_id_list in baseline_hvac_sys_type_ids_dict_b.items() if hvac_b.id in hvac_id_list), None)`